runtime/vimrc_example.vim

```diff
@@ -80,3 +80,8 @@ if !exists(":DiffOrig")
   command DiffOrig vert new | set bt=nofile | r ++edit # | 0d_ | diffthis
                  \ | wincmd p | diffthis
 endif
+
+" Prevent that the langmap option applies to characters that result from a
+" mapping.  If unset (default), this may break plugins (but it's backward
+" compatible).
+set langnoremap
```

Comment:
:+1:  glad to see you didn't add the `if has('langmap') && exists('+langnoremap')` check